packages/components/src/components/Paragraph/Paragraph.tsx

```diff
@@ -45,20 +41,20 @@ const getParagraphStyles = (
 
 /** A Paragraph is a block of text. */
 const Paragraph = React.forwardRef<HTMLParagraphElement, ParagraphProps>(
-  ({ children, marginBottom = "space70", size = "medium", ...props }, ref) => {
+  ({ children, size = "medium", ...props }, ref) => {
     return (
-      <Text.p
+      <Box.p
```

Comment:
Do you know the difference between using Text or Box?
Why can't we continue using Text in here?

Reply:
I got an error that comes from the type since it was accepting only text props (fontWeight, color) instead of accepting more props like margin, padding, etc